Apps/W1/HybridBaseDeployment/app/src/codeunits/CloudMigReplicateDataMgt.Codeunit.al

```diff
@@ -71,6 +71,46 @@ codeunit 40021 "Cloud Mig. Replicate Data Mgt."
         exit(CanBeIncluded);
     end;
 
+    procedure CheckAndIncludeExcludeTableListFromCloudMigration(TableIDList: List of [Integer]; NewReplicateData: Boolean)
+    var
+        Company: Record Company;
```

Comment:
This is incorrect as we will be updating all companies and not only the companies that are included in the migration. 
Maybe company name should become a part of the signature to add the table?

Reply:
We can either drop this function or do it in a different way. The idea was to be able to say - I want same packages of tables for all companies included in the migration.

Reply:
same as above, lets start with basic functions and add overloads where we feel that they are needed.

Reply:
"Per table" functions added.